packages/frontend/src/pages/PlaygroundCreate.svelte

```diff
@@ -55,7 +56,7 @@ async function submit() {
   // disable submit button
   submitted = true;
   try {
-    trackingId = await studioClient.requestCreatePlayground(playgroundName, model);
+    trackingId = await studioClient.requestCreatePlayground(playgroundName, model, systemPrompt ?? '');
```

Comment:
```suggestion
    trackingId = await studioClient.requestCreatePlayground(playgroundName, model, systemPrompt ?? undefined);
```

Reply:
That was my first attempt, but the problem I can see is that systemPrompt can be either undefined or an empty string, depending on if the user lets the textarea untouched, or edits its content and removes it. I wanted to have only one value (the empty string) to indicate the systemPrompt is empty. But as the two values are handled correctly on the other side, we can do like this if you prefer

Reply:
I think that the empty value should not be allowed, and be replaced with `undefined`